views/breach-detail.hbs

```diff
@@ -18,7 +18,7 @@
       <div class="col-8 overview">
         {{> breach-detail-content-group overview}}
         {{#if delayedReporting }}
-          <a class="text-link" href="#{{delayedReporting.href}}">
```

Comment:
Should we hardcode the slug, or should we change the `href: "delayedReporting"` to `href: "delayed-reporting"`? Or should we just delete the unused `href`?

https://github.com/mozilla/blurts-server/blob/23af592f389da106e205a774e1ccfa2535cb3d37/template-helpers/breach-detail.js#L218-L224